app/Project.swift

```diff
@@ -127,6 +130,15 @@ let project = Project(
                 .target(name: "TuistErrorHandling"),
             ]
         ),
+        .target(
+            name: "TuistNoora",
```

Comment:
this should eventually make it back to `tuist/noora`. But for now, let's keep iterating on the SwiftUI version of Noora locally